app/src/main/kotlin/org/fossify/gallery/fragments/ViewPagerFragment.kt

```diff
@@ -92,6 +94,16 @@ abstract class ViewPagerFragment : Fragment() {
 
     fun getPathToLoad(medium: Medium) = if (context?.isPathOnOTG(medium.path) == true) medium.path.getOTGPublicPath(context!!) else medium.path
 
+    private fun getResolution(medium: Medium, file: File): String? {
+        if (medium.name.endsWith(".jxl")) {
```

Comment:
The case should be ignored here. 

```suggestion
        if (medium.name.endsWith(".jxl", ignoreCase = true)) {
```